zipline/finance/performance/tracker.py

```diff
@@ -125,12 +126,20 @@ def __init__(self, sim_params, trading_calendar, env):
                     create_first_day_stats=True
                 )
 
+        try:
+            benchmark = self.asset_finder.lookup_symbol(
```

Comment:
Doing our own ad-hoc symbol resolution here seems like an easy way for the benchmarks to get out of sync across different zipline subsystems. For example, I think the benchmark resolution is supposed to respect `set_symbol_lookup_date`.  Is there a reason we can't just pass the benchmark asset itself at construction time for this class?